Homework 13 feature/ansible 4

DmitriySh opened the pull request:
Homework 13 by Vagrant/Ansible/Molecule/Testinfra

DmitriySh commented:
Специально сделал отдельную папку /vagrant, чтобы оставить в /ansible работу инкрементально и с ролями в том числе, для истории. В /vagrant ansible только через роли работает. Очень много всего накапливалось в /ansible и выглядело не очень удобно для восприятия.

Artemmkin commented:
дак а где ты меняешь провижининг в пакере не вижу?

DmitriySh commented:
Дополнил.

Artemmkin commented:
Подразумевалось, что при провижининге пакером будет выполняться не вся роль, а только те таски, которые были до использования роли, для этого мы помечали таски тегами (см. [ссылку](https://www.packer.io/docs/provisioners/ansible.html#extra_arguments))

DmitriySh commented:
Мне нужно указать с ролью те теги, которые должны быть выполнены?

Artemmkin commented:
нужно указать в пакере тегированные таски, которые ты хочешь выполнить

Artemmkin commented:
т.е. передать тег в пакере

DmitriySh commented:
добавил `"extra_arguments": "--tags=install"`

```
    googlecompute: TASK [app : Install ruby and rubygems and required packages] *******************
    googlecompute: changed: [default] => (item=[u'ruby-full', u'ruby-dev', u'build-essential'])
    googlecompute:
    googlecompute: TASK [app : Install Ruby bundler] **********************************************
    googlecompute: changed: [default]
    googlecompute:
    googlecompute: TASK [app : add unit file for Puma] ********************************************
    googlecompute: changed: [default]
    googlecompute:
    googlecompute: TASK [app : add config for DB connection] **************************************
    googlecompute: changed: [default]
    googlecompute:
    googlecompute: TASK [app : enable puma] *******************************************************
    googlecompute: changed: [default]
    googlecompute:
    googlecompute: RUNNING HANDLER [app : reload puma] ********************************************
    googlecompute: changed: [default]
```

Artemmkin commented:
да, теперь выглядит ок. Уже одобрил, поэтому можно мерджить
